.github/workflows/tag.yml

```diff
@@ -0,0 +1,27 @@
+name: Tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version (e.g. v0.1.0)'
+        required: true
+
+jobs:
+  create-tag:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Fetch History
+        run: git fetch --prune --unshallow
+
+      - name: Create Tag
+        run: make -j2 tag
+        env:
```

Comment:
I think we optionally allow a commit hash to be specified in current job: https://github.com/crossplane/crossplane/blob/1ddb11225ea70c4ac425b50d13ead1d8fc7ed507/Jenkinsfile.tag#L32

Reply:
Looks like github gives us the commit hash we can use if default if desired also: https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#workflow_dispatch

Reply:
I omitted this intentionally, under the assumption that it was a feature we never used. Do you think it's useful/necessary?

Reply:
Does `make tag` use latest commit on branch if not specified? I am guessing yes, but just checking that we don't need to provide our own default.

Reply:
I believe it does - in the previous `Jenkinsfile.tag` it looks like COMMIT_HASH would be empty by default.